platform/ios/detect.py

```diff
@@ -26,13 +26,16 @@ def get_opts():
     return [
         ("vulkan_sdk_path", "Path to the Vulkan SDK", ""),
         (
-            "IOS_TOOLCHAIN_PATH",
-            "Path to iOS toolchain",
-            "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain",
+            # Example: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain
+            "APPLE_TOOLCHAIN_PATH",
```

Comment:
Build scripts should be updated accordingly to use new names - https://github.com/godotengine/godot-build-scripts/blob/6cc1418b6bdde424e4cb708658fe7f00aff1953c/build-ios/build.sh

Reply:
Submitted https://github.com/godotengine/godot-build-scripts/pull/113